protocol/0057-TRAN-transfers.md

```diff
@@ -264,11 +267,16 @@ message CancelTransfer {
   - If I have enough funds to pay transfer and fees, the transfer happens.
   - If I do not have enough funds to pay transfer and fees, the transfer is cancelled.
   - The fees are being paid into the infrastructure pool
+  - The transfer fee discount is correctly applied with network parameter`transfer.feeDiscountDecay`(<a name="0057-TRAN-014" href="#0057-TRAN-014">0057-TRAN-014</a>)
+  - The fee-free transfer amount is accessible while API (<a name="0057-TRAN-017" href="#0057-TRAN-017">0057-TRAN-017</a>)
 - As a user I can do a transfer from any of the valid accounts (I control them and they're a valid source), and fees are taken from the source account when the transfer is executed (when `transfer amount * transfer.fee.factor > transfer.fee.maxQuantumAmount * quantum`). (<a name="0057-TRAN-011" href="#0057-TRAN-011">0057-TRAN-011</a>)
-  - The fee cost is correctly calculated using the network parameter `transfer.fee.Maxfactor`
+  - The fee cost is correctly calculated using the network parameter
   - If I have enough funds to pay transfer and fees, the transfer happens.
   - If I do not have enough funds to pay transfer and fees, the transfer is cancelled.
   - The fees are being paid into the infrastructure pool
+  - The transfer fee discount is correctly applied with network parameter`transfer.feeDiscountDecay`(<a name="0057-TRAN-015" href="#0057-TRAN-015">0057-TRAN-015</a>)
```

Comment:
I think we want a couple of cases here:
1. no fees paid / generated from trading ==> full amount paid
2. sufficient taker fees paid in previous epoch ==> no amount paid on transfer
3. some taker fees paid in previous epoch ==> one transfer is free but next transfer is not (because there weren't so many trading fees generated)
4. some taker fees paid in previous epoch, few epochs pass so that the amount decays but still enough for a free transfer
5. some taker fees paid in previous epoch, many epochs pass, counter is `0` because it decayed below minimum ==> transfer fees charged.
6. all of the above but the party making transfer is never the aggressor - they are always passive but the fees are attributed to them for transfers.